shared/src/commonMain/kotlin/org/hyperskill/app/profile/view/BadgeDetailsViewState.kt

```diff
@@ -0,0 +1,18 @@
+package org.hyperskill.app.profile.view
+
+import org.hyperskill.app.badges.domain.model.BadgeKind
+import org.hyperskill.app.badges.domain.model.BadgeRank
+
+data class BadgeDetailsViewState(
+    val kind: BadgeKind,
+    val rank: BadgeRank,
+    val title: String,
+    val formattedRank: String,
+    val imageSource: String?,
```

Comment:
Here you should return `BadgesViewState.BadgeImage` (locked or remote), like in the badges grid view.